rocketpy/mathutils/function.py

```diff
@@ -13,13 +13,22 @@
 import matplotlib.pyplot as plt
 import numpy as np
 from scipy import integrate, linalg, optimize
+from bisect import bisect_left
```

Comment:
The bisect, a native python function should be imported before 3rd party libraries. That's for convention and readability.

Do you run `isort` before submitting a PR?
You can do that by running `make isort` in your terminal, our simply `isort .`